From bf7b58a2471a07111f8022c0176f45ee5dc5fe71 Mon Sep 17 00:00:00 2001
From: Mark McLoughlin <markmc@redhat.com>
Date: Thu, 16 Apr 2009 14:21:35 +0000
Subject: [PATCH] qemu -drive takes format= not fmt=

Seems like a simple typo - it has been "format=" since the flag
was introduced, but we added it as "fmt=".

(cherry picked from commit 9fa79000ecc883c699a6cb1ce7f00c34881bc8fe)

Fedora-patch: libvirt-0.6.2-qemu-drive-format.patch
---
 src/qemu_conf.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/qemu_conf.c b/src/qemu_conf.c
index f36c927..6f9e610 100644
--- a/src/qemu_conf.c
+++ b/src/qemu_conf.c
@@ -1135,7 +1135,7 @@ int qemudBuildCommandLine(virConnectPtr conn,
                 disk->device == VIR_DOMAIN_DISK_DEVICE_DISK)
                 virBufferAddLit(&opt, ",boot=on");
             if (disk->driverType)
-                virBufferVSprintf(&opt, ",fmt=%s", disk->driverType);
+                virBufferVSprintf(&opt, ",format=%s", disk->driverType);
 
             if (disk->cachemode) {
                 const char *mode =
-- 
1.6.2.5

